app/src/main/res/layout/layout_empty_transactions.xml

```diff
@@ -1,63 +1,84 @@
+      android:layout_height="match_parent"
+      >
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingBottom="55dp"
+        >
+      <TextView
+          android:id="@+id/no_transactions_text"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="@dimen/half_large_margin"
+          android:gravity="center"
+          android:text="@string/no_transactions_yet"
+          app:layout_constraintBottom_toTopOf="@id/transactions_empty_screen_animation"
+          app:layout_constraintEnd_toStartOf="@+id/guideline2"
+          app:layout_constraintStart_toStartOf="@+id/guideline"
+          app:layout_constraintTop_toTopOf="parent"
+          />
 
-  <TextView
-      android:id="@+id/action_learn_more"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginTop="16dp"
-      android:text="@string/activity_transactions_learn_more"
-      android:textColor="@color/colorAccent"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@id/action_air_drop"
-      style="@style/Widget.AppCompat.Button.Borderless"
-      />
-  <androidx.constraintlayout.widget.Guideline
-      android:id="@+id/guideline"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:orientation="vertical"
-      app:layout_constraintGuide_begin="80dp"
-      />
-  <androidx.constraintlayout.widget.Guideline
-      android:id="@+id/guideline2"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:orientation="vertical"
-      app:layout_constraintGuide_end="80dp"
-      />
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+      <com.airbnb.lottie.LottieAnimationView
+          android:id="@+id/transactions_empty_screen_animation"
+          android:layout_width="140dp"
+          android:layout_height="140dp"
+          app:layout_constraintBottom_toTopOf="@id/earn_bonus_text"
+          app:layout_constraintEnd_toStartOf="@+id/guideline2"
+          app:layout_constraintStart_toStartOf="@+id/guideline"
+          app:layout_constraintTop_toBottomOf="@id/no_transactions_text"
+          app:lottie_autoPlay="true"
+          app:lottie_enableMergePathsForKitKatAndAbove="true"
+          app:lottie_loop="true"
+          app:lottie_rawRes="@raw/transactions_empty_screen_animation"
+          />
+
+      <TextView
+          android:id="@+id/earn_bonus_text"
+          android:layout_width="0dp"
+          android:layout_height="wrap_content"
+          android:gravity="center"
+          android:text="@string/gamification_home_body"
```

Comment:
Adding default string containing replacement tag is not a good if string is not replaced.